docs/contracts/interface-ids.md

```diff
@@ -29,7 +29,7 @@ const LSP0_INTERFACE_ID = INTERFACE_IDS.LSP0ERC725Account;
 If you are trying to import the `INTERFACE_IDS` within a Hardhat Typescript project, use the following import syntax:
 
 ```ts
-import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts/dist/constants.cjs.js';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts/constants';
```

Comment:
i tihnk we still have a bunch of these bad imports in the repo

Reply:
We did. And they are already included in this PR (3x) 👍🏻 Playground does not have any `/dist/` anymore.